Covinator/Covinator/Pages/PfizerVaccination.cshtml.cs

```diff
@@ -21,92 +19,112 @@ public PfizerVaccination(ILogger<PfizerVaccination> logger)
             _logger = logger;
         }
 
+        public IList<string> JurisdictionsList = new List<string>();
+
         public void OnGet()
         {
-            using (var webClient = new WebClient())
+            try
             {
-                string pfizerData = webClient.DownloadString("https://data.cdc.gov/resource/saz5-9hgg.json");
-                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("pfizerSchema.json"));
-                JArray jsonArray = JArray.Parse(pfizerData);
-                IList<string> validationEvents = new List<string>();
-                var currentWeek1 = jsonArray[0].ToArray();
-                var currentWeek = currentWeek1[1].ToString();
```

Comment:
Removed this logic and replaced it -
var currentWeek = jsonArray.FirstOrDefault()?["week_of_allocations"]?.Value<DateTime>().ToShortDateString();

to fetch only the date from current string. Before this the value in frontend was sent as "weak_of_allocations":"2021-12-04 12:00:00", now the value is "12-Apr-2021"